lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1029,6 +1029,9 @@ fc_extras
         try:
             iris.unit.as_unit(attr_units)
         except ValueError:
+        # formatting a unicode string causes an exception. Test for that.
+            if isinstance(attr_units, unicode):
+                attr_units = attr_units.encode('ascii', errors='backslashreplace')
```

Comment:
a test is failing because of this change 

previously the 'unknown' unit string was a unicode, e.g.
```
(('invalid_units', u'wibble'),)
```

following the change, this is encoded as:
```
(('invalid_units', 'wibble'),)
```

please either alter the behaviour (better) or the test result (if you can justify it)
/net/home/h04/itmh/scitools/iris/lib/iris/tests/results/netcdf/netcdf_units_0.cml